ui/src/app/kvm/components/KVMHeaderForms/AddLxd/AddLxd.test.tsx

```diff
@@ -86,7 +86,7 @@ describe("AddLxd", () => {
   });
 
   it(`shows the authentication form if the user has generated a certificate for
-    the LXD VM host`, () => {
+    the LXD KVM host`, () => {
```

Comment:
I updated the test names and comments as well so we're using the same language everywhere.